packages/config/config/devices/0x0113/lrm-as.json

```diff
@@ -36,18 +44,62 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "No",
+					"label": "Use Start Level",
 					"value": 0
 				},
 				{
-					"label": "Yes",
+					"label": "Ignore Start Level",
+					"value": 1
+				}
+			]
+		},
+		"3": {
+			"label": "LED Indication",
+			"description": "The WD500Z-1 will flicker its LED when it is transmitting to any of its groups.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "LED ON - Switch ON",
+					"value": 0
+				},
+				{
+					"label": "LED ON - Switch OFF",
```

Comment:
The description doesn't describe what is happening in the options.

Reply:
I will update and re-submit. I copied this line for line from the existing WD500z handler so I may update that one if I get this one approved. I suspect it also doesn't meet standards.